Student/Student.xcodeproj/xcshareddata/xcschemes/Student.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "1010"
+   LastUpgradeVersion = "1000"
```

Comment:
Is that intentional? I think this was just updated to 1010 in https://github.com/instructure/canvas-ios/pull/36